AFQ/tests/test_api.py

```diff
@@ -138,6 +138,46 @@ def create_dummy_bids_path(n_subjects, n_sessions):
     return bids_dir
 
 
+@pytest.mark.nightly3
+def test_AFQ_custom_tract():
+    """
+    Test whether AFQ can use tractography from
+    custom_tractography_bids_filters
+    """
+    tmpdir = nbtmp.InTemporaryDirectory()
+    afd.organize_stanford_data(path=tmpdir.name)
+    afd.fetch_stanford_hardi_tractography()
+    bids_path = op.join(tmpdir.name, 'stanford_hardi')
+
+    bundle_names = ["SLF", "ARC", "CST", "FP"]
+
+    # move subsampled tractography into bids folder
+    os.rename(
+        op.join(
+            op.expanduser('~'),
+            'AFQ_data',
+            'stanford_hardi_tractography',
+            'tractography_subsampled.trk'),
+        op.join(
+            tmpdir.name,
+            'stanford_hardi',
+            'derivatives',
+            'vistasoft',
```

Comment:
Does the tractography have to reside in the 'dmriprep' derivatives folder?  Can it be in another folder (i.e., generated by some other pipeline)?

Reply:
Good point. Scope should be specified by the user filters instead.

Reply:
I made this change in "make scope a user choice" commit. Does this look like a good solution?